### PR TITLE
Updating cli docs for octopus.migrator.exe

### PR DESCRIPTION
**Pull request by Octobob**
An automated update of the command line docs for octopus.migrator.exe.

Created by TeamCity project 'Automated Documentation Updates::Update CLI Docs', build 49

### CODE REVIEW

File: docs/api-and-integration/octopus.migrator.exe-command-line/import.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: Imports data from an Octopus 3.0+ export directory
+---
+
+Imports data from an Octopus 3.0+ export directory
+
+**import options**
+
+```text
+
+```
```
Comment (wordlee) [inline docs/api-and-integration/octopus.migrator.exe-command-line/import.md]: The options appear to be missing on this one.

File: docs/api-and-integration/octopus.migrator.exe-command-line/partial-export.md [hunk 2]
```diff
@@ -9,8 +9,56 @@ This command exports configuration data to a directory filtered by a single proj
 
 Usage:
 
-```bash
-Octopus.Migrator partial-export [<options>]
+```text
+Usage: octopus.migrator partial-export [<options>]
```
Comment (Waldo000000) [inline docs/api-and-integration/octopus.migrator.exe-command-line/partial-export.md]: These options look right to me :+1: But we've now got a repeated section beneath (under `Partial export options`).

**Comment by wordlee** [comment]
It looks like we have a newer PR for this. Are we happy to close this one?

**Comment by Waldo000000** [comment]
Yup @wordlee I'll close this old one now :+1:

**Pull request closed without merging**